### DISCUSSION TITLE
Zorin OS not working

### DISCUSSION THREAD

**Discussion by siminayazi** [community/community#135163] [Programming Help] [answered]
### Body

i install zorin os and it dosent work sound what should i do?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by aradfarahani** [comment, marked as answer]
Hello @siminayazi 
I'm sorry to hear you're having trouble with sound on Zorin OS. Here are some steps you can try to resolve the issue:

1. **Check Sound Settings**:
   - Open the Zorin Menu and go to **Settings**.
   - Select the **Sound** panel from the left sidebar.
   - Ensure the correct **Output Device** is selected and the volume is turned up.

2. **PulseAudio Volume Control**:
   - If you're using Zorin OS Lite, PulseAudio Volume Control should already be installed. Open it from the Zorin Menu → **Sound & Video** → **PulseAudio Volume Control**.
   - If not, install it by opening the Terminal and running:
     ```bash
     sudo apt install pavucontrol
     ```
   - After installation, open PulseAudio Volume Control and check the settings under the **Output Devices** and **Configuration** tabs.

3. **Restart the Audio Server**:
   - Open the Terminal and run:
     ```bash
     systemctl --user restart pulseaudio
     ```
   - After restarting the audio server, you may need to restart any applications that were using audio.

4. **Install Missing Codecs**:
   - Sometimes, missing codecs can cause sound issues. Open the Terminal and run:
     ```bash
     sudo apt install ubuntu-restricted-extras
     ```

5. **Check Hardware Connections**:
   - Ensure your speakers or headphones are properly connected to the correct audio jack.

If these steps don't resolve the issue, you might want to check the [Zorin Help page](https://help.zorin.com/docs/hardware/set-up-sound/)[ for more detailed troubleshooting](https://help.zorin.com/docs/hardware/set-up-sound/).
